plugins/hydra_ax_sweeper/example/conf/config.yaml

```diff
@@ -26,6 +26,8 @@ hydra:
         # An Epoch is defined as a batch of trials executed in parallel
         max_epochs_without_improvement: 10
 
+      is_errorless: true
+
```

Comment:
Can you add a comment here explaining what this does? (this is an example people will look at).